Migrate from junit.framework to org.junit

**Summary:** 
Assert in `junit.framework` has been deprecated so we should migrate to `org.junit`

Link for help https://stackoverflow.com/questions/24013210/assert-in-junit-framework-has-been-deprecated-what-next-to-use

**Would you like to work on the issue?**
No

Hi, I have changed the import statements from junit.framework to org.junit. Should I create a separate branch and push the code?
When I tried building the project, I get this error "Caused by: org.codehaus.groovy.control.MultipleCompilationErrorsException: startup failed:" , could you please help me out on building the project? Thanks a lot :)

--- a/app/src/test/kotlin/fr/free/nrw/commons/description/DescriptionEditActivityUnitTest.kt
+++ b/app/src/test/kotlin/fr/free/nrw/commons/description/DescriptionEditActivityUnitTest.kt
@@ -19,8 +19,8 @@ import fr.free.nrw.commons.description.EditDescriptionConstants.WIKITEXT
 import fr.free.nrw.commons.settings.Prefs
 import fr.free.nrw.commons.upload.UploadMediaDetail
 import fr.free.nrw.commons.upload.UploadMediaDetailAdapter
-import junit.framework.Assert.assertEquals
 import org.junit.Assert
+import org.junit.Assert.assertEquals
 import org.junit.Before
 import org.junit.Test
 import org.junit.runner.RunWith

--- a/app/src/test/kotlin/fr/free/nrw/commons/review/ReviewControllerTest.kt
+++ b/app/src/test/kotlin/fr/free/nrw/commons/review/ReviewControllerTest.kt
@@ -13,9 +13,9 @@ import fr.free.nrw.commons.R
 import fr.free.nrw.commons.TestAppAdapter
 import fr.free.nrw.commons.TestCommonsApplication
 import fr.free.nrw.commons.delete.DeleteHelper
-import junit.framework.Assert.assertEquals
-import junit.framework.Assert.assertNotNull
+import org.junit.Assert.assertEquals
 import media
+import org.junit.Assert.assertNotNull
 import org.junit.Before
 import org.junit.Test
 import org.junit.runner.RunWith

--- a/app/src/test/kotlin/fr/free/nrw/commons/upload/UploadMediaPresenterTest.kt
+++ b/app/src/test/kotlin/fr/free/nrw/commons/upload/UploadMediaPresenterTest.kt
@@ -15,7 +15,7 @@ import io.github.coordinates2country.Coordinates2Country
 import io.reactivex.Observable
 import io.reactivex.Single
 import io.reactivex.schedulers.TestScheduler
-import junit.framework.TestCase.assertEquals
+import org.junit.Assert.assertEquals
 import org.junit.Before
 import org.junit.Test
 import org.junit.runner.RunWith

